### PR TITLE
Show disabled queues to course staff

### PR DESCRIPTION
**Pull request by IrisLiu-00**
# Description

Update today page so that course staff can see both open and non open queue cards. Students still only see open queues.

I also updated the loading skeleton for the queue card to match the new card design. 

Closes # (https://github.com/sandboxnu/office-hours/issues/874)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe how you tested this PR (both manually and with tests)
Provide instructions so we can reproduce. 

- [ ] manually
- [ ] need to update cypress tests :(

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code where needed 
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots
image

### CODE REVIEW

File: packages/app/pages/course/[cid]/today.tsx [hunk 1]
```diff
@@ -72,6 +73,9 @@ export default function Today(): ReactElement {
   const role = useRoleInCourse(Number(cid));
   const { course, mutateCourse } = useCourse(Number(cid));
 
+  const parts = course?.queues && partition(course?.queues, (q) => q.isOpen);
+  const sortedQueues = parts && parts[0].concat(parts[1]); // put open queues first
```
Comment (VKong6019) [inline packages/app/pages/course/[cid]/today.tsx:77]: hehe, what if we used the vanilla Array `sort` method with a comparator. so smth like this (assuming you want open queues before the non open ones)
```suggestion
  const sortedQueues = [...course?.queues].sort(({isOpen: a}, {isOpen:b}) => Number(b) - Number(a)); // put open queues first
```

Comment (VKong6019) [reply packages/app/pages/course/[cid]/today.tsx:77]: _take my black magic_

File: packages/app/components/Today/OpenQueueCard.tsx [hunk 2]
```diff
@@ -274,11 +278,24 @@ export default OpenQueueCard;
...
-      <Skeleton paragraph={{ rows: 2 }} />
-      <Row justify="space-between" align="bottom">
-        <Skeleton.Avatar size={96} />
+    <PaddedCard
+      headStyle={{
+        background: "#25426C",
+        color: "#FFFFFF",
+        borderRadius: "6px 6px 0 0",
+      }}
+      className={"open-queue-card"}
```
Comment (VKong6019) [inline packages/app/components/Today/OpenQueueCard.tsx:287]: if this file now considers closed queues, should we rename the file from `OpenQueueCard` to just `QueueCard` for slightly more clarity?

**Review by VKong6019 — COMMENTED** [review]
cypress looks scary

File: packages/app/pages/course/[cid]/today.tsx (hunk 1 above)
Comment (IrisLiu-00) [reply packages/app/pages/course/[cid]/today.tsx:77]: ok i had a `sort` first but then i changed it for some reason thinking this was simpler lol. ill put it back 👍

**Review by VKong6019 — APPROVED** [review]
Yar

**Comment by dfarooq610** [comment]
unfortunately, merge conflicts

**Comment by dfarooq610** [comment]
The only _slight nit_ I have is I think that we should throw professor queues at the absolute bottom for TAs (since they'll never use them)

I'd imagine:

| TA               |   Professor   |
| ----------- | ----------- |
| Open      |  Open   |
| Closed Reg Queues   | Closed Prof Queues     |
| Closed Prof Queues   | Closed Reg Queues    |

**Review by tiingweii-shii — APPROVED** [review]
:D

**Pull request merged**